tt_metal/host_api.hpp

```diff
@@ -318,7 +318,7 @@ std::shared_ptr<Buffer> CreateBuffer(const InterleavedBufferConfig &config, Devi
 std::shared_ptr<Buffer> CreateBuffer(const InterleavedBufferConfig &config, SubDeviceId sub_device_id);
 
 /**
-*  Creates a pre-allocated sharded DRAM or L1 buffer with the global allocator on device
+*  Creates a pre-allocated sharded DRAM or L1 buffer with the global allocator on the device. Sharded buffers use parallel processing to allow independent access to different data sets. For more information about buffers visit the Tenstorrent GitHub: [Buffers info on GitHub](https://github.com/tenstorrent/tt-metal/tree/main/tt_metal/impl/buffers).
```

Comment:
"use parallel processing to allow independent access to different data sets" is very inaccurate.

Sharded buffers store data locally per core. Best way of using sharded data is if each core operates on that local data, so no NoC data transfers are needed, but I don't see how "buffers use parallel processing".